config/routes.rb

```diff
@@ -591,4 +591,5 @@
   end
 
   match 'host_wizard' => 'react#index', :via => :get
+  resources :user_preferences
```

Comment:
```suggestion
  resources :user_preferences, only: [:index, :create]
```
Looks like only `:index` and `:create` are defined in the controller.

Reply:
So this can be removed?

Reply:
@waldenraines We wouldn't want to define routes for controller methods that don't exist (`:show`, `:destroy`, etc.)  So adding `only: [:index, :create]` to the end of the line will only give you `:index` and `:create` routes, and prevents the other routes from being defined.  See https://guides.rubyonrails.org/routing.html#restricting-the-routes-created